frontend/app/routes/_gcweb-app.tsx

```diff
@@ -54,6 +55,7 @@ function ApplicationLayout({ children }: { children?: ReactNode }) {
         {children}
         <PageDetails />
       </main>
+      <SessionTimeout />
```

Comment:
I wonder if it makes more sense to put this in `root.tsx`, right below the toaster?

Reply:
updated